migrations/versions/a5183973c3e9_.py

```diff
@@ -0,0 +1,23 @@
+"""Change SQLAlchemy Boolean bit(1) to bool
+
+Revision ID: a5183973c3e9
+Revises: 2e0d2e02a721
+Create Date: 2023-08-10 15:51:13.537000
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = 'a5183973c3e9'
+down_revision = '2e0d2e02a721'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.alter_column('regression_test', 'active', existing_type=sa.Boolean(), type_=sa.Boolean(), nullable=False)
```

Comment:
```suggestion
    op.alter_column('regression_test', 'active', existing_type=sa.Boolean(), type_=sa.Boolean(), nullable=False)
```

This seems wrong to me compared to the downgrade step.

Reply:
Can you please explain why it is wrong? (I have tested this on spdev as well...)

Reply:
Both the existing type and the type are the same values, "sa.Boolean". I think existing type should be "sa.dialects.mysql.BIT(1)".

Reply:
Oh yes, actually it is a optional arg, required for MYSQL but it had no effect on the Command generated, it would be something like `ALTER TABLE regression_test MODIFY active bool;`